app/src/main/res/layout/activity_main.xml

```diff
@@ -4,37 +4,152 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/ic_launcher_background"
     tools:context=".MainActivity">
 
+<!--  COMMENTED OUT BECAUSE BUTTONS ARE NO LONGER USEFUL /AND/ NOT SURE IF VIDEO IS GOOD IDEA
```

Comment:
And again here. I mean, it's debatable whether commiting commented code is a good practice or not, but I don't think Lisa would like it.

Reply:
Yeah, garbage in my code is not good.